astropy/coordinates/builtin_frames/utils.py

```diff
@@ -32,10 +32,14 @@
 _DEFAULT_PM = (0.035, 0.29)*u.arcsec
 
 _IERS_HINT = """
-If you need enough precision such that this matters (~<10 arcsec), you can download the latest IERS predictions by doing:
-from astropy.utils.data import download_file
-from astropy.utils import iers
-iers.IERS.iers_table = iers.IERS_A.open(download_file(iers.IERS_A_URL, cache=True))"""
+If you need enough precision such that this matters (~<10 arcsec), you can
+download the latest IERS predictions by running:
+
+    >>> from astropy.utils.data import download_file
+    >>> from astropy.utils import iers
+    >>> iers.IERS.iers_table = iers.IERS_A.open(download_file(iers.IERS_A_URL, cache=True))
```

Comment:
For this matter, I think the IERS_A object should be able to recognize URL filenames, and incorporate the download_file stuff automatically.  That could be done as a separate PR though.